sessions/css-grid/image-board/css/styles.css

```diff
@@ -0,0 +1,57 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
+    Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+}
+
+/* 
+- Use grid-template areas to place the images as shown in the challenges file
+- Assign an area to each image
+*/
```

Comment:
This part should be written inside a readme file. You could take a look at the readme files of the objects and arrays challenges https://github.com/neuefische/web-exercises/pull/65.